src/manifest.rs

```diff
@@ -83,3 +92,80 @@ pub(crate) fn manifest(uri: &str, resource_type: &str, settings: Option<String>)
 
     Ok(())
 }
+
+fn get_policy_title_from_cli_or_metadata(
+    policy_title: Option<String>,
+    metadata: &Metadata,
+) -> Option<String> {
+    policy_title.or_else(|| {
+        metadata
+            .annotations
+            .as_ref()
+            .unwrap_or(&HashMap::new())
+            .get(KUBEWARDEN_ANNOTATION_POLICY_TITLE)
+            .map(|s| s.to_string())
+    })
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn mock_metadata_with_no_annotations() -> Metadata {
+        Metadata {
```

Comment:
To some extent it's a pity that we don't implement `Default` for `Metadata`. It would make this tests much more concise. However, conceptually it probably doesn't make sense to derive it just for the sake of making usage easier. So yeah, nothing to do here :)